app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -143,4 +141,14 @@ def api_authorization_token
     }]
     User.current.jwt_token!(expiration: 4.hours.to_i, scope: scope)
   end
+
+  def find_organization
+    from_id = Organization.authorized(:view_organizations).find(params['organization_id']) if params['organization_id'].present?
+    from_id || User.current.default_organization || User.current.my_organizations.first
```

Comment:
```suggestion
    return Organization.authorized(:view_organizations).find(params['organization_id']) if params['organization_id'].present?
    User.current.default_organization || User.current.my_organizations.first
```
If the parameter exists, then the first line is the only one that is executed. The second line is the default, in case the org_id parameter was not defined in the request.